src/components/lotes/LotesGroup.jsx

```diff
@@ -57,24 +102,33 @@ class LotesGroup extends Component {
   }
 
   render() {
-    const { allLotes, selectedLotes, removeLoteFromGroup } = this.props;
-    const filteredLotes = this.filterLotes(allLotes, selectedLotes);
+    const { allLotes, selectedLotes } = this.props;
```

Comment:
does it matter that the lotes come from props not state? I guess this container is rerendered each time the lotes change?